### ISSUE TITLE
Event-Creators can edit own Event

### ISSUE THREAD

**Issue by lujobi** [amiv-eth/amivapi#361]
Since errors happen quite frequently, it would be great if event creators could edit their own events.

**Comment by NotSpecial** [comment]
Thank you for your input. I have a follow-up question.

What is the extent of permissions the event creater should have?
Only modifying the event? Or also modifying the event participants?

**Comment by lujobi** [comment]
For me it would be enough if the creator can only modify the event. Or do you mean the manual selection? I think that should be done together with a vorstand.

**Comment by hermannsblum** [comment]
For clarification: Who is an event creator?
And how is his permission different from somebody who can edit an event?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lujobi** [comment]
The one who created the event, so it can be a "normal" user, whose event got confirmed.

**Comment by hermannsblum** [comment]
So if I get this right you are talking about the function to „propose“ events with a link send to a vorstand?

In particular, thinking about our discussion here https://github.com/amiv-eth/amivapi/issues/353, did you consider adding these people into the group that can edit events?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lujobi** [comment]
Sorry for the late answer. With the last website everyone who had access to the admin link could see participants and edit the event. At the moment only the vorstand and maybe some other groups can do so. This is quite tedious especially for the Kulturvorstände. It would be great if the one who "proposes" the event would have the permissions to edit the event and see the participants.

**Comment by NotSpecial** [comment]
Thanks for the clarification. Would it also be acceptable to "propose" changes to the event, similar to how an event is proposed?

That might not be to hard to implment, right, @hermannsblum ?

**Comment by hermannsblum** [comment]
that would be possible. 
In the end however, it would really make sense to have Event Moderators, this seems to be much better fit to how the Kulturteam operates.
Might be a bigger chance, but would create a better tool for users.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by NotSpecial** [comment]
Fair enough. I think it should be more or less straightforward.

I think we would probably go with a system like the group moderator. The only issue I can think of is that the event moderator could replace him/herself with someone else .. but I think that's about as much a problem as changing the event text to something obscene. And both have to be more or less monitored by the team, nothing we can do about it.

Long story short: Seems to be a good issue to tackle on the coding weekend.

**Comment by NotSpecial** [comment]
Event moderators are now part of the API, as of #377.
